lms/djangoapps/bulk_email/models.py

```diff
@@ -107,7 +107,7 @@ def get_users(self, course_id, user_id=None):
         staff_qset = CourseStaffRole(course_id).users_with_role()
         instructor_qset = CourseInstructorRole(course_id).users_with_role()
         staff_instructor_qset = (staff_qset | instructor_qset)
-        enrollment_qset = User.objects.filter(
+        enrollment_query = models.Q(
```

Comment:
I think `Q` objects are the more semantic way of doing it, and we can achieve your abstraction level with them the same way we would with `kwargs`. Are `Q` objects very expensive?